### PR TITLE
Be able to ignore some CWE

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: .github/workflows/audit.yaml [hunk 1]
```diff
@@ -31,21 +31,25 @@ jobs:
       - run: |
           for file in $(find -name requirements.txt)
           do
-            (cd $(dirname ${file}) && safety check --full-report --file=requirements.txt)
+            echo Audit ${file}
+            (cd $(dirname ${file}) && safety check --full-report --ignore=$(cat safety-ignore|true) \
```
Comment (arnaud-morvan) [inline .github/workflows/audit.yaml:35]: Seems --ignore can be specified many times, but should not support comma separated list of ids.

Comment (arnaud-morvan) [reply .github/workflows/audit.yaml:35]: https://github.com/pyupio/safety#--ignore--i

**Comment by sbrunner** [comment]
pyenv looks to be used my pipenv to install alernate python version

File: .github/workflows/audit.yaml (hunk 1 above)
Comment (sbrunner) [reply .github/workflows/audit.yaml:35]: Yes, in the safety-ignore I should add comma separated list of ids :-)

**Pull request merged**